Source/lighting.h

```diff
@@ -21,8 +21,8 @@ namespace devilution {
 
 #define MAXLIGHTS 32
 #define MAXVISION 32
-/** 16 light levels + infravision + stone curse + red for pause/death screen */
-#define LIGHTSIZE (19 * 256)
+/** 16 light levels */
+#define LIGHTSIZE 16

```

Comment:
```suggestion
/** @brief Number of supported light radiuses (first radius starts with 0) */
constexpr size_t NumLightLadiuses = 16;
```

Improvement question (not directly related):
Should we add asserts in `DoLighting` and `AddLight` that the radius is not high enough?

Reply:
Since they are two different things I added the new one in https://github.com/diasurgical/devilutionX/pull/6012/commits/af48fb234673545237df7fd3711f03f1fba161cc :)